src/verilog/verilog_compiler.cpp

```diff
@@ -164,7 +164,9 @@ void CompileVerilog(const std::vector<std::string_view> &inputs_v_full_paths,
 	// https://stackoverflow.com/questions/31434380/what-is-a-good-template-yosys-synthesis-script
   // Yosys::Pass::call(&yosys_design, "synth"); -> this is sort of the commands below:
   // CHECK: the default with "./yosys_exe" > "help synth"
+  
   Yosys::Pass::call(&yosys_design, "hierarchy -check");
+  Yosys::Pass::call(&yosys_design, "hierarchy -check -top main");
```

Comment:
revert `-top main` -> breaks CI and generic circuits

Reply:
Did not find a another way to fix to fix: LFSR_comb top module.
try to change the order in circuit_lib.cpp but it does not work.